eth_utils/conversions.py

```diff
@@ -31,6 +24,10 @@ def to_hex(value=None, hexstr=None, text=None):
     Trims leading zeros, as defined in:
     https://github.com/ethereum/wiki/wiki/JSON-RPC#hex-value-encoding
     """
+    from .hexidecimal import (
+        add_0x_prefix,
+        encode_hex,
+    )
```

Comment:
Seems like there's a circular dependency now between `.conversions` and `.hexidecimal`. Is there a better way to handle it? Reorganization of some kind?

Reply:
Yes, normally I try first to see if there is a way to re-organize the utils.  If it's not clear how to do that, I just inline the imports into the function bodies to remove the circular dependency at module load time.